src/views/HashtagPostView/index.tsx

```diff
@@ -1,8 +1,10 @@
 import { useEffect, useState } from 'react'
 
+import { ChevronDownIcon, ChevronUpIcon } from '@chakra-ui/icons'
```

Comment:
import { FaChevronUp, FaChevronDown } from 'react-icons/fa'

Reply:
I fixed. Thank you for your suggestions